packages/falcon-client/src/server.js

```diff
@@ -22,6 +23,11 @@ export function Server({ App, clientApolloSchema, bootstrap, webpackAssets, load
 
   const publicDir = process.env.PUBLIC_DIR;
   const router = new Router();
+
+  if (config.graphqlProxy) {
+    graphqlProxy(config, router, port);
```

Comment:
I think it should be possible to do it in that way:
```suggestion
    graphqlProxy(config, router, port);
    router.all(...graphqlProxy(config, port))
```